src/Panel/panel.scss

```diff
@@ -60,12 +60,12 @@
   }
 
   > .panel-header {
-    overflow: hidden;
     color: $white;
     background-color: $charcoal;
 
     .expand-icon {
       font-size: 20px;
+      padding: 0 10px;
```

Comment:
I used padding before change `flexBasis` to `basis` and I forgot to remove it :see_no_evil: